ocaml/xenopsd/xc/device.ml

```diff
@@ -52,8 +52,6 @@ module Profile = struct
   let all = [Qemu_none; Qemu_upstream_compat; Qemu_upstream]
 
   module Name = struct
-    let qemu_none = "qemu-none"
```

Comment:
It looks like there is a missing entry  in `Profile.of_string`. I think it'll be possible to run without Qemu once we have support for PVH (for which I had an old branch of mine). Instead of dropping qemu_none perhaps we can add it to of_string instead? There *is* a `Qemu_none` backend module there already.